site/content/en/docs/tasks/run/multikueue/_index.md

```diff
@@ -1,7 +1,27 @@
 ---
-title: "Run Workloads in MultiKueue"
-weight: 2
+title: "Using MultiKueue"
+linkTitle: "MultiKueue"
+weight: 8
 date: 2024-09-25
 description: >
   The workloads that support MultiKueue.
 ---
+
+This page explains how to run tasks in MultiKueue environment.
+
+For more details about the MultiKueue check the concepts section for a [MultiKueue overview](/docs/concepts/multikueue/). 
+
+
+## Before you begin
+
+Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+
+## Running tasks in MultiKueue environment
+
+Any job compatible with Kueue in a single-cluster environment can be seamlessly deployed in a multi-cluster setup using MultiKueue.
```

Comment:
Yes, we try to make it as easy as possible, but might not be seamless in practice. For example, if the worker cluster is in another zone or region, then the user needs to solve the storage complications (copy input data, synchronize outputs, etc.).  I would just drop this sentence for now.